iina/PrefKeyBindingViewController.swift

```diff
@@ -155,101 +144,74 @@ class PrefKeyBindingViewController: NSViewController, PreferenceWindowEmbeddable
     mappingController.remove(sender)
     NotificationCenter.default.post(Notification(name: .iinaKeyBindingChanged))
   }
+  
+  // Check whether or not a new config file with provided filename should be created.
+  // Returns the path of the new config if could be created; nil otherwise
+  private func checkNewConfigFile(with filename: String) -> String? {
+    // Check if the name is empty
+    guard !filename.isEmpty else {
+      Utility.showAlert("config.empty_name", sheetWindow: self.view.window)
+      return nil
+    }
+    
+    // Check if there already exists a config which has the same name
+    guard KC.userConfigs[filename] == nil && KC.defaultConfigs[filename] == nil else {
+      Utility.showAlert("config.name_existing", sheetWindow: self.view.window)
+      return nil
+    }
+    
+    // Check if there exists a config file with the same filename
+    let filePath = Utility.userInputConfDirURL.appendingPathComponent(filename + ".conf").path
+    if fm.fileExists(atPath: filePath) {
+      Utility.quickAskPanel("config.file_existing", sheetWindow: self.view.window) { result in
+        if result == .alertFirstButtonReturn {
+          NSWorkspace.shared.activateFileViewerSelecting([URL(fileURLWithPath: filePath)])
+        }
+      }
+      return nil
+    }
```

Comment:
> I also remove the file deletion code because maybe its too dangerous to remove user files with "OK" or an enter hit in a seemingly unrelated alert window (the user want to create a config, but end up with deleting their file), let alone that part of code was never been executed before due to the erroneous usage.

At least we should tell the user how to delete this file (where it is) in the dialog.

Reply:
The default action (OK button or enter) will bring up the Finder and show the conflicting config file